ui/components/Header.js

```diff
@@ -321,10 +330,45 @@ function K8sContextMenu({
 
   const handleKubernetesDelete = async (name, connectionID) => {
     let responseOfDeleteK8sCtx = await deleteCtxtRef.current.show({
-      title: `Delete ${name} context ?`,
-      subtitle: `Are you sure you want to delete ${name} cluster from Meshery?`,
+      title: `Delete Kubernetes connection?`,
+      subtitle: (
+        <>
+          <Typography variant="body">
+            {' '}
+            Are you sure you want to delete Kubernetes connection &quot;{name}&quot; and associated
+            credential?
+          </Typography>
+          <Typography
+            variant="body2"
+            sx={{ textAlign: 'left', marginTop: '1rem', cursor: 'pointer' }}
+          >
+            <details>
+              <summary>
+                <strong>What does this mean?</strong>
+              </summary>
+              <div>
```

Comment:
It would be nice to have a slightly different background color inside of the details. I tried adding one, but...

Reply:
...I could use some help, @aabidsofi19

Reply:
i will update this .